### DISCUSSION TITLE
trying to understand user dictionaries

### DISCUSSION THREAD

**Discussion by thane-k** [florisboard/florisboard#1943] [Q&A]
trying to add, for example, terms that use diacritics in their letters, by adding the term with diacritics in the dictionary, and have tried adding the term without diacritics as the shortcut, as the language code, and also as both together; in the hopes that the term with, can be entered by typing it without, but neither seem to work. the shortcut function was imagined to do the trick? according to personal interpretation at least
(to give a better example if needed, café with the accented e, to be entered by typing cafe)

also hoping to be able to enter in special symbols, e.g. alt-codes (☺☻♥♦♣♠•◘○ etc.) into the dictionary, similar to the aforementioned terms with diacritics, so that they too can be entered by inputting something perhaps a little different.

are either of these possible?

**Comment by patrickgold** [comment]
User dictionaries are a thing which are somewhat broken right now (especially because word suggestions is one big construction zone), so you understood the dictionaries correctly (also the shortcuts) but the implementation of it in the suggestions algorithm is just missing.

I do plan to fix user dictionaries sometime in 0.4 though!

**Reply by shary998** [reply]
can you kindly tell me when will you start working on version 0.4?

**Reply by tsiflimagas** [reply]
As already mentioned in multiple places, work on 0.4 has already started and the first alpha is released already. To answer this in advance, there's no ETA.
